doc/modules/lda_qda.rst

```diff
@@ -70,10 +70,10 @@ the class conditional distribution of the data :math:`P(X|y=k)` for each class
 and we select the class :math:`k` which maximizes this conditional probability.
 
 More specifically, for linear and quadratic discriminant analysis,
-:math:`P(X|y)` is modelled as a multivariate Gaussian distribution with
```

Comment:
"modelled" doesn't seem wrong to me. It's just not US English. British and Canadian English seem to like it as "modelled". Otherwise LGTM, thanks @zdgriffith!